### ISSUE TITLE
Two minor issues

### ISSUE THREAD

**Issue by Swartz27** [xrOxygen/xray-oxygen#93]
1) When compiling the source manually by downloading a zip of the current main repo, the code fails to compile due to unresolved externals regarding NVAPI (I'm referring to the renderers of course).

2) Unfortunately with the past several "releases" that I try, I can't even get the game to start. The log says that things regarding "BUY_MENU" are not there. I can get more specific by giving a detailed error log if you'd like.

I also want to offer my most sincere apologizes for what a complete and total prick I was to you guys in the past. You guys are doing an excellent job, and my criticisms from before are really just nitpicks: you guys have shown that you are VERY competent programmers. Things go astray with rogue commits here and there but that's to be expected and it always gets fixed.

Basically: I feel I owe you guys. Not because I'm using your code (yet), but because of my previous hostility (I promise it won't happen again). I'm not concerned about my own mod, I'm thinking, if you guys let me, I can help you with some shader specific things that it looks like there has been a few issues with. I was taught a lot by an industry pro about graphics/shaders so I'm willing to share everything that I know.

Anyway; tl;dr: I've been a dick and I'm sorry, I just poked my head in to give you guys a heads up about those issues and also offer help if I can (just keep me away from C++, I am the worst C++ programmer in the universe).

**Comment by ForserX** [comment]
@Swartz27: How are the successes with the a port into
 the English version?

**Comment by Swartz27** [comment]
@ForserX Easy peasy :)  Everything is working properly. Great work to you and the rest of the team as well. This is some really sweet stuff.

**Comment by ForserX** [comment]
@Swartz27, if you the fixed starting on eng version - you can make PR?
